components/sidebar/index.tsx

```diff
@@ -174,7 +182,7 @@ const Sidebar = ({ data }: { data: Data }): JSX.Element => {
 				},
 			}));
 		}
-	}, [walletAddress, data.wallets]);
+	}, [walletAddress, data.wallets, timeframe]);
```

Comment:
👏

Reply:
Took me a minute here but I realized that we certainly would want to run this whenever each of these changes